src/compute/common/alignment.rs

```diff
@@ -35,10 +35,23 @@ pub(crate) fn compute_alignment_offset(
             AlignContent::Center => free_space / 2.0,
             AlignContent::Stretch => 0.0,
             AlignContent::SpaceBetween => 0.0,
-            AlignContent::SpaceAround => (free_space / num_items as f32) / 2.0,
-            AlignContent::SpaceEvenly => free_space / (num_items + 1) as f32,
+            AlignContent::SpaceAround => {
+                if free_space >= 0.0 {
+                    (free_space / num_items as f32) / 2.0
```

Comment:
Can `num_items` ever be 0? I wonder if we need to account for division by zero problems here

Reply:
It cannot. Because this function is called on a per-item (or per-group-of-items) basis. So if there are no items, then it will be skipped :)